tests/components/homekit/test_homekit.py

```diff
@@ -1,12 +1,12 @@
 """Tests for the HomeKit component."""
-from unittest.mock import patch, ANY, Mock
+from unittest.mock import patch, ANY, Mock, MagicMock, PropertyMock
 
 import pytest
 
 from homeassistant import setup
 from homeassistant.components.homekit import (
     generate_aid, HomeKit, STATUS_READY, STATUS_RUNNING,
-    STATUS_STOPPED, STATUS_WAIT)
+    STATUS_STOPPED, STATUS_WAIT, MAX_DEVICES)
```

Comment:
Just some small requests, to keep the style: Can you order the import alphabetically?